datacenter/install/linux.md

```diff
@@ -20,7 +20,7 @@ EE have a minimum of:
 
 Also, make sure the hosts are running one of these operating systems:
 
-* CentOS 7.1 or 7.2
+* CentOS 7.4
```

Comment:
IIRC, CentOS itself even refers to this as `CentOS 7` (as it is a rolling release model, so basically there is no `.1`, `.2`, `.3`)

Reply:
For sure. @thaJeztah, check out this PR: #4678.